src/libcore/macros.rs

```diff
@@ -15,7 +15,7 @@ macro_rules! panic {
         panic!("explicit panic")
     );
     ($msg:expr) => ({
-        static _MSG_FILE_LINE: (&'static str, &'static str, u32) = ($msg, file!(), line!());
+        const _MSG_FILE_LINE: (&'static str, &'static str, u32) = ($msg, file!(), line!());
```

Comment:
This is a case where the use of a `static` is actually done as an optimization. This guarantees that the data here is translated at most once and makes calling the underlying function efficient. (e.g. this should remain a `static`)